pkg/apis/imageregistry/v1/types.go

```diff
@@ -208,6 +208,8 @@ type ImageRegistryConfigStorageSwift struct {
 	DomainID  string `json:"domainID,omitempty"`
 	Tenant    string `json:"tenant,omitempty"`
 	TenantID  string `json:"tenantID,omitempty"`
+	// The name of the secret containing the openstack clouds.yaml file
+	CloudsSecret *corev1.SecretReference `json:"cloudsSecret"`
```

Comment:
We don't want to have other namespaces: they require additional informers, changing RBAC rules, etc. It's better to use corev1.SecretKeySelector.

But we also need to bootstrap ourselves. When the operator gets installed to a new cluster, it should inspect ConfigMap kube-config/cluster-config-v1 and configure the storage automatically. Will the operator get this secret from the installer or will it somehow to create it? How will it distinguish secrets that are created automatically and that are provided by a user?